web/src/assets/locales/de.json

```diff
@@ -139,6 +139,7 @@
         "cancel": "Abbrechen",
         "cancel_success": "Pipeline abgebrochen",
         "canceled": "Dieser Schritt wurde abgebrochen.",
+        "deploy": "Deploy",
```

Comment:
This should be in en.json

Reply:
This was a leftover, it is in en.json lin 233.

This was a hard one , thanks for your patience